easybuild/easyconfigs/b/Beast2/Beast2-2.4.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = "Tarball"
+
+name = 'Beast2'
```

Comment:
@pescobar as mentioned in #3165, I'd stick to `Beast`, unless you think it's useful to have both `Beast` v1.x and v2.x to be loaded at the same time